### PR TITLE
Update fp-011-locus-of-authority.md

### PR DESCRIPTION
**Pull request by nataled**
Add brief TOC

### CODE REVIEW

File: principles/fp-011-locus-of-authority.md
Comment (matentzn) [inline principles/fp-011-locus-of-authority.md:6]: @erik-whiting before merging make sure you fix this to relative links!

Comment (matentzn) [reply principles/fp-011-locus-of-authority.md:6]: (Also wait for at least one EWG review before merging!)

**Review by cstoeckert — CHANGES_REQUESTED** [review]
Section tags need to be added in order for the TOC links to work. Also won’t need full URL if do that just the tag. Minor point - might make the tag to Examples rather than Counter-Examples.

**Comment by nataled** [comment]
Yes, please do relative links (I tried that, but couldn't verify that it worked).

**Comment by nataled** [comment]
Agree with @cstoeckert, the section tag for Examples/Counter-Examples should be #Examples. Or maybe it needs to be lowercase. I cannot get consistent behavior from the example I followed, so it's difficult to know the best way to do it.

**Comment by nataled** [comment]
That's what I tried first, but it only worked for the single-word headings, so I tried doing that for all. This is why I'm not part of the TWG!

**Comment by nataled** [comment]
It also seems that they have to be lowercase?

File: principles/fp-011-locus-of-authority.md
Comment (erik-whiting) [reply principles/fp-011-locus-of-authority.md:6]: @matentzn I think it should be the PR author's responsibility to use the correct links

**Comment by nataled** [comment]
I agree with @erik-whiting, and would be happy to do it, but nothing I've tried really worked, so I'm not sure how to do the links properly. So far the only thing that worked was what I did, and even that only worked for the single-word sections (which is why I removed 'and Requirements', for example). So if someone can guide me with one example (say, for 'Recommendations and Requirements), I'll take care of the rest.

**Comment by nataled** [comment]
Having now seen Chris's comment on github rather than in the email, I have the examples I need. Will revise accordingly.

**Review by cstoeckert — APPROVED** [review]

File: principles/fp-011-locus-of-authority.md
Comment (cthoyt) [inline principles/fp-011-locus-of-authority.md:6]: we should really switch to using bootstrap-toc for autogenerating these. future work i might be able to take care of

**Comment by cthoyt** [comment]
I request that we make more descriptive titles for PRs - this one is not possible to figure out what it does without reading the diff

**Comment by nataled** [comment]
@cthoyt sorry, I should have spelled out 'table of contents' instead of writing 'TOC'. I'm not sure how much more descriptive I could get than that.

**Comment by cthoyt** [comment]
I meant on "Update fp-011-locus-of-authority.md" no worries though

**Comment by nataled** [comment]
@cthoyt Oh I see. In the past when I've changed that title to something descriptive, I realize that I no longer know which file is being changed, so I've taken to just leaving it alone and adding info to the 'extra' description field. I'll try combining in the future (hopefully there will be enough space).

**Comment by nataled** [comment]
Now that this has been merged and on the main site, I can see that the page does not render the way the Edit -> Preview led me to believe. If you go to https://obofoundry.org/principles/fp-011-locus-of-authority.html and look at the table at the top, you'll notice that there are no separators between the section titles. If you click the Edit button, and immediately go to 'Preview', you'll see that there are clear separations (put in using a space, a pipe, and another space). Any idea how to get that to appear on the main page?

**Comment by erik-whiting** [comment]
> Now that this has been merged and on the main site ...

@nataled The preview is a markdown preview only and isn't a good representation of what will be seen on website (which is served via Jekyll, a system that has its own interpretation of markdown). If you want a high fidelity preview of the work you've done, you'll have to pull the repository and run Jekyll locally, either in Docker or directly on your own system.

**Pull request merged**
